components/Layout.tsx

```diff
@@ -52,7 +58,7 @@ export default function Layout({ children, mainClassName, metaTitle, whiteBg }:
         <main className={
           classnames(mainClassName, 'z-10 xl:rounded-xl py-4 mx-auto')
         }>
-          <header className={classnames('w-full bg-white fixed top-0 z-[170] shadow-xl drop-shadow-lg')}>
+          <header className={classnames('w-full bg-white fixed top-0 z-[170] shadow-xl',  { 'h-16': !isLandingPage, 'drop-shadow-lg': isLandingPage, 'shadow-sm': !isLandingPage })}>
```

Comment:
<img width="1508" alt="Screenshot 2024-02-23 at 02 05 41" src="https://github.com/json-schema-org/website/assets/40007659/e072127a-ab9f-4254-a1ba-3f663c0bac55">

This is what I see. I think we need to reduce the space on top and try to center the elements vertically to have the same space between the top and the bottom. We can also reduce the size of the logo.

We should check the mobile versión too.